server/gokbg3/grails-app/services/wekb/AutoUpdatePackagesService.groovy

```diff
@@ -1008,9 +1008,9 @@ class AutoUpdatePackagesService {
                 }
 
                 if (minimumKbartStandard.size() != countMinimumKbartStandard) {
-                    log.info("KBART file has in header not the minimumKbartStandard: ${minimumKbartStandard}")
+                    log.info("KBART file does not have one or any of the headers: ${minimumKbartStandard}")
```

Comment:
is missing at least one of the following headers

Reply:
Miriam gab die Vorgabe.